app/controllers/candidate_interface/course_choices/review_interruption_controller.rb

```diff
@@ -7,6 +7,17 @@ def show
         @application_choice = current_application.application_choices.find(params[:application_choice_id])
         @word_count = current_application.becoming_a_teacher.scan(/\S+/).size
       end
+
+      def continue_without_editing_path
```

Comment:
Can I suggest prefixing this with `x_` so it's clear that it is not a named route

Reply:
Also, this probably shouldn't be in the controller. It's not an action, and it's not even being called in the controller.

Reply:
I thought was too overkill to be a component. Do you have a suggestion where to put?

Reply:
I moved to a component. The good thing is that I added specs for it.